src/app/[locale]/page.tsx

```
@@ -16,42 +19,36 @@ export default function Home() {
+            <span className="text-muted-foreground text-sm font-medium">{t("hero.status")}</span>
           </div>
 
           <h1
             className="text-foreground animate-reveal mb-8 text-6xl font-bold tracking-tight md:text-8xl"
             style={{ animationDelay: "0.1s" }}
           >
-            Bouwen aan het <br />
-            <span className="from-primary bg-linear-to-r to-purple-400 bg-clip-text text-transparent">
-              Moderne Web.
-            </span>
+            <span dangerouslySetInnerHTML={{ __html: t.raw("hero.title") }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)